app/jobs/delete_missed_reservations_job.rb

```diff
@@ -0,0 +1,18 @@
+class DeleteMissedReservationsJob < ActiveJob::Base
+  queue_as :default
+
+  def perform
+    return if AppConfig.first.blank? || AppConfig.first.res_exp_time.blank?
+    time = AppConfig.first.res_exp_time
+    missed_reservations =
```

Comment:
While we're rewriting these tasks, I feel like we should take the opportunity to refactor where possible for readability to promote the use of short private methods with easy-to-understand names (effectively, [extract method](http://refactoring.com/catalog/extractMethod.html)). It might seem a bit heavy-handed but I feel like it will improve our code readability moving forward.